Livecheckables/druid.rb

```diff
@@ -1,4 +1,4 @@
 class Druid
-  livecheck :url   => "http://druid.io/downloads.html",
+  livecheck :url   => "https://druid.io/downloads.html",
```

Comment:
```suggestion
  livecheck :url   => "https://druid.apache.org/downloads.html",
```

This redirected when I tested it, so we should update the URL to avoid the redirection.